Moblin/Various/Model.swift

```diff
@@ -5274,11 +5330,16 @@ extension Model: WCSessionDelegate {
         guard let value = data as? Bool else {
             return
         }
-        DispatchQueue.main.async {
-            if value {
-                self.startStream()
-            } else {
-                self.stopStream()
+        if database.watch!.remoteControl! {
+            remoteControlAssistantSetStream(on: value)
+            sendIsLiveToWatch(isLive: value)
+        } else {
+            DispatchQueue.main.async {
+                if value {
+                    self.startStream()
+                } else {
+                    self.stopStream()
+                }
```

Comment:
The whole if-else should be in main async block, right?

Reply:
done